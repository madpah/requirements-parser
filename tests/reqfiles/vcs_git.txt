-e git+git://git.myproject.org/MyProject#egg=MyProject
-e git+https://git.myproject.org/MyProject#egg=MyProject
-e git+ssh://git.myproject.org/MyProject#egg=MyProject
-e git://git.myproject.org/MyProject.git@master#egg=MyProject
-e git://git.myproject.org/MyProject.git@v1.0#egg=MyProject
-e git://git.myproject.org/MyProject.git@da39a3ee5e6b4b0d3255bfef95601890afd80709#egg=MyProject

git+git://git.myproject.org/MyProject#egg=MyProject
git+https://git.myproject.org/MyProject#egg=MyProject
git+ssh://git.myproject.org/MyProject#egg=MyProject
git://git.myproject.org/MyProject.git@master#egg=MyProject
git://git.myproject.org/MyProject.git@v1.0#egg=MyProject
git://git.myproject.org/MyProject.git@da39a3ee5e6b4b0d3255bfef95601890afd80709#egg=MyProject
