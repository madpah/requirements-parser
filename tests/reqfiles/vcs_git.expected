[
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "git+git://git.myproject.org/MyProject", 
  "extras": [], 
  "vcs": "git", 
  "path": null, 
  "line": "-e git+git://git.myproject.org/MyProject#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "git+https://git.myproject.org/MyProject", 
  "extras": [], 
  "vcs": "git", 
  "path": null, 
  "line": "-e git+https://git.myproject.org/MyProject#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "git+ssh://git.myproject.org/MyProject", 
  "extras": [], 
  "vcs": "git", 
  "path": null, 
  "line": "-e git+ssh://git.myproject.org/MyProject#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "git://git.myproject.org/MyProject.git", 
  "extras": [], 
  "vcs": "git", 
  "path": null, 
  "line": "-e git://git.myproject.org/MyProject.git@master#egg=MyProject", 
  "specs": [], 
  "revision": "master"
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "git://git.myproject.org/MyProject.git", 
  "extras": [], 
  "vcs": "git", 
  "path": null, 
  "line": "-e git://git.myproject.org/MyProject.git@v1.0#egg=MyProject", 
  "specs": [], 
  "revision": "v1.0"
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "git://git.myproject.org/MyProject.git", 
  "extras": [], 
  "vcs": "git", 
  "path": null, 
  "line": "-e git://git.myproject.org/MyProject.git@da39a3ee5e6b4b0d3255bfef95601890afd80709#egg=MyProject", 
  "specs": [], 
  "revision": "da39a3ee5e6b4b0d3255bfef95601890afd80709"
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "git+git://git.myproject.org/MyProject", 
  "extras": [], 
  "vcs": "git", 
  "path": null, 
  "line": "git+git://git.myproject.org/MyProject#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "git+https://git.myproject.org/MyProject", 
  "extras": [], 
  "vcs": "git", 
  "path": null, 
  "line": "git+https://git.myproject.org/MyProject#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "git+ssh://git.myproject.org/MyProject", 
  "extras": [], 
  "vcs": "git", 
  "path": null, 
  "line": "git+ssh://git.myproject.org/MyProject#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "git://git.myproject.org/MyProject.git", 
  "extras": [], 
  "vcs": "git", 
  "path": null, 
  "line": "git://git.myproject.org/MyProject.git@master#egg=MyProject", 
  "specs": [], 
  "revision": "master"
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "git://git.myproject.org/MyProject.git", 
  "extras": [], 
  "vcs": "git", 
  "path": null, 
  "line": "git://git.myproject.org/MyProject.git@v1.0#egg=MyProject", 
  "specs": [], 
  "revision": "v1.0"
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "git://git.myproject.org/MyProject.git", 
  "extras": [], 
  "vcs": "git", 
  "path": null, 
  "line": "git://git.myproject.org/MyProject.git@da39a3ee5e6b4b0d3255bfef95601890afd80709#egg=MyProject", 
  "specs": [], 
  "revision": "da39a3ee5e6b4b0d3255bfef95601890afd80709"
 }
]
